solr/packaging/test/test_create.bats

```diff
@@ -34,8 +34,31 @@ teardown() {
   assert_output --partial "Created new core 'COLL_NAME'"
 }
 
+@test "create_core works" {
+  run solr start
+  run solr create_core -c COLL_NAME
+  assert_output --partial "Created new core 'COLL_NAME'"
+}
+
 @test "create for cloud mode" {
   run solr start -c
   run solr create -c COLL_NAME
   assert_output --partial "Created collection 'COLL_NAME'"
 }
+
+@test "ensure -p port parameter is supported" {
+  solr start -p ${SOLR2_PORT}
+  solr assert --not-started http://localhost:${SOLR_PORT} --timeout 5000
+  solr assert --started http://localhost:${SOLR2_PORT} --timeout 5000
+  
+  run solr create -c COLL_NAME -p ${SOLR2_PORT}
+  assert_output --partial "Created new core 'COLL_NAME'"
+}
+
+@test "ensure -V verbose parameter is supported" {
+  solr start 
+  solr assert --started http://localhost:${SOLR_PORT} --timeout 5000
+  
+  run solr create -c COLL_NAME -V
+  assert_output --partial "Created new core 'COLL_NAME'"
```

Comment:
Should not this assert a string that is only printed in verbose mode?

Reply:
Good question...    I think for core, you got this message, and for collection, you got a different dump (that was more like the api response).   I changed it so that both core and collection output a short message like this, and then if you add the `-v` then you ALSO see the api responses...   Does this work for you.

Reply:
Sure, I just expected the assert to fail without the `-V`, which it won't now since the text asserted is always printed?

Reply:
I finally understand what you mean, and am looking at it and then will merge.